src/interpreter/crypto.rs

```diff
@@ -42,12 +43,16 @@ impl<S> Interpreter<S> {
     pub(crate) fn keccak256(&mut self, a: Word, b: Word, c: Word) -> Result<(), RuntimeError> {
```

Comment:
The same here, more descriptive argument names=)

Reply:
created an issue to fix this broadly
https://github.com/FuelLabs/fuel-vm/issues/220